src/jv_instrument.cpp

```diff
@@ -290,6 +290,7 @@ void JournalViewer::setJournal(Journal* jrnl)
 	}
 
 	// New journal data has been loaded (hopefully), so must update limits and unique lists
+	storeFilters();
```

Comment:
Regarding the update issue, do we just need to call `retrieveFilters()` after the `resetFilters()` call below?

Reply:
no, retrieveFilters just seems to update the limits on the lockable filters. Calling versions of this function doesn't seem to change anything.
filterRunData retrieves the actual filter values within itself and is called but still presents the issue.